src/NHibernate/Impl/SessionImpl.cs

```diff
@@ -2099,12 +2099,6 @@ public override void AfterTransactionBegin(ITransaction tx)
 					Interceptor.AfterTransactionBegin(tx);
 				}
 			}
-
-			if (!_transactionCoordinatorShared)
-				foreach (var dependentSession in ConnectionManager.DependentSessions)
-				{
-					dependentSession.AfterTransactionBegin(tx);
-				}
```

Comment:
That is for this kind of changes I would like to understand the rational. This will cause dependent interceptors to lack the begin transaction event and only receive the end ones. Without any comment telling why is this, it sounds to me as a bug.